docs/content/installation/csi_ug_requirements.md

```diff
@@ -37,33 +37,60 @@ Perform these steps for each worker node in Kubernetes cluster to prepare your e
+   **Important:** The `99-ibm-attach.yaml` configuration file overrides any files that already exist on your system. Only use this file if the files mentioned are not already created. <br />If one or more have been created, edit this YAML file, as necessary.
 
-   Apply the yaml file.
+   Apply the YAML file.
 
    `oc apply -f 99-ibm-attach.yaml`
+
+3. Configure storage system connectivity.
+
+    1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
+
+    2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
     
-3. If needed, enable support for volume snapshots (FlashCopy® function) on your Kubernetes cluster.
+4. (Optional) If planning on using volume snapshots (FlashCopy® function), enable support on your Kubernetes cluster.
 
    For more information and instructions, see the Kubernetes blog post, [Kubernetes 1.20: Kubernetes Volume Snapshot Moves to GA](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/).
 
    Install both the Snapshot CRDs and the Common Snapshot Controller once per cluster.
 
-   The instructions and relevant yaml files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
+   The instructions and relevant YAML files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
 
-4. Configure storage system connectivity.
+5. (Optional) If planning on using volume replication (remote copy function), enable support on your Kubernetes cluster.
+
+    Install the following replication CRDs once per cluster.
+
+    ```
+    curl -O https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.1.0/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml
+    kubectl apply -f ./replication.storage.openshift.io_volumereplicationclasses.yaml
+    
+    curl -O https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.1.0/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml
+    kubectl apply -f ./replication.storage.openshift.io_volumereplications.yaml
+    ```
+
+6. (Optional) To use CSI Topology, any nodes in the cluster must have the following labels that introduce topology awareness:
+      - `topology.kubernetes.io/region`
+      - `topology.kubernetes.io/zone`
+
+      **Important:** These labels must be found on the nodes in the cluster **before** installing the IBM® block storage CSI driver. If these are not labeled before installation, the CSI driver will not be topology aware.
+
+      ```
+      $> kubectl get nodes -o=jsonpath='{range .items[*]}[{.metadata.name}, {.metadata.labels}]{"\n"}{end}' | grep --color "topology.block.csi.ibm.com"
```

Comment:
should we have an explanation for the purpose of this command?

Reply:
is this not the purpose?
" If these are not labeled before installation, the CSI driver will not be topology aware."

Reply:
this command just shows the current labels in the cluster (filtered by the required prefix).
IMO there's no need for this

Reply:
I can remove but please confirm. I put this in based off of the information given to me. @ArbelNathan  Please confirm that I can remove this command or if I should put in a different one, what it should be.

Reply:
@oriyarde what is the problem with the command?

Reply:
I don't see the point in maintaining it. its output contains irrelevant labels and it doesn't perform any require change

Reply:
ok, so let's remove it, what are you suggesting to put instead for the user to know that he "done it right"?

Reply:
according to [this](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#step-one-attach-label-to-the-node), the user can use `kubectl get nodes --show-labels` or `kubectl describe node "nodename"`.
but I don't think it matters - IMO, we should not maintain this info

Reply:
we don't even maintain the command for labeling a node

Reply:
DONE